routes/project/location/load_balancer.rb

```diff
@@ -79,10 +79,12 @@ class Clover
           authorize("LoadBalancer:edit", lb)
           algorithm, health_check_endpoint = typecast_params.nonempty_str!(%w[algorithm health_check_endpoint])
           src_port, dst_port = typecast_params.pos_int!(%w[src_port dst_port])
+          cert_enabled = typecast_params.bool!("cert_enabled")
```

Comment:
This breaks backwards compatibility:

```suggestion
          cert_enabled = typecast_params.bool("cert_enabled")
```

If parameter is not submitted, we should assume the setting is unchanged.